tools/topology/topology2/cavs-sdw.conf

```diff
@@ -85,6 +85,7 @@ Define {
 IncludeByKey.PLATFORM {
 	"mtl"	"platform/intel/mtl.conf"
 	"lnl"	"platform/intel/lnl.conf"
+	"ptl"	"platform/intel/ptl.conf"
```

Comment:
Probably a dumb question sorry: why is this file called CAVS-sdw.conf?

Reply:
because we started using it on cavs... it's meaningless, only the sdw part matters.